helm/node-exporter-app/templates/daemonset.yaml

```diff
@@ -67,7 +67,11 @@ spec:
         - '--collector.xfs'
 
         - '--no-collector.btrfs'
+        {{- if eq (include "provider" .) "azure" }}
+        - '--collector.diskstats'
```

Comment:
In order to have easily understandable manifest on installations, I'd propose to split this into two conditionals. One that sets the diskstats above in `--collector` switch segment and one that sets `no-collector` here in case provider is not Azure.

Reply:
makes sense

Reply:
Done